### ISSUE TITLE
# loss = (loss - args.gamma)**2 

### ISSUE THREAD

**Issue by moreUPL** [GuanZihan/GNN_backdoor_detection#1]
May I ask if this **Ltrap** corresponds to the line annotated above, but this line has been commented out in the code and **gamma** has not been used elsewhere?

**Comment by GuanZihan** [comment]
1. Yes, you are right. This is the code for Ltrap loss. We comment this because of the contribution experiment in the paper. You can directly uncomment this code. 
2. Yes, gamma is only used here since gamma is only a value for constructing the loss function.
Sorry for the inconvenience!
